sql/src/main/java/org/apache/druid/sql/calcite/rule/DruidRules.java

```diff
@@ -115,13 +115,13 @@ public static List<RelOptRule> rules(PlannerContext plannerContext)
       retVal.add(DruidOuterQueryRule.WINDOW);
     }
 
-    if (plannerContext.featureAvailable(EngineFeature.UNNEST)) {
```

Comment:
Am confused by this, so the unnest should still be behind this flag?

Reply:
Yes.

Reply:
Won't that be confusing because to enable unnest, the user has to specify the flag through the query context? That was one of the main reasons we are trying to get rid of the flag. We introduced the flag as unnest was an experimental feature at that time. Is the outlook still to treat this as an experimental feature?

Reply:
Apologies if its confusing. I meant keep the engine feature but remove the "query context flag" .

Reply:
Makes sense, I have added it back